### DISCUSSION TITLE
Help for youtube transcript. How to use from Open WeBui

### DISCUSSION THREAD

**Discussion by peuportier** [open-webui/open-webui#2135] [Q&A]
I'm curious about using Open Web UI to extract video transcripts from YouTube videos. Is this process similar to reading content from a website, such as #example.com, or is there an alternative method? Thank you for your assistance.
Thanks you, Wish the team a great day.

**Comment by justinh-rahb** [comment]
@peuportier Yes, it is similar to how the web RAG works, in that it's just converting the transcript (or HTML in the case of web) to vector embeddings and doing RAG on them.

**Comment by peuportier** [comment]
@justinh-rahb  Thanks you for your answer. Very good job
